src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -346,12 +346,6 @@ private static String getValue(final String arg) {
 
   public static void setupLogging(GameMode gameMode) {
```

Comment:
Somewhere down the rabbit hole I think there was a dependency that required these updates. Otherwise perhaps so, moot point I think.